packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { lazy, Suspense, useContext, useMemo, useState } from 'react';
```

Comment:
i don't see the component test?

Reply:
Tests for this component are done inside the `layout` test file (as we discussed during september 5th react sharing meeting)